DataDefinitions/BlueprintMaterials.cs

```diff
@@ -20,7 +20,7 @@ public BlueprintMaterials(List<MaterialAmount> materials)
 
         static BlueprintMaterials()
         {
-            string data = Net.DownloadString("http://api.eddp.co/_blueprintrequirements");
+            string data = Net.DownloadString("https://raw.githubusercontent.com/EDCD/EDDP/master/blueprintrequirements.json");
```

Comment:
Please revise to use the server URL as specified in Constants.cs, like this:
`string data = Net.DownloadString(Constants.EDDI_SERVER_URL + "blueprintrequirements.json");`